### ISSUE TITLE
Windows integration testing

### ISSUE THREAD

**Issue by johannesduesing** [delphi-hub/delphi-registry#58]
**Describe the problem**
While developing the docker support and the database integration for the instance registry, we occasionally ran into some problems regarding docker for windows and slick in combination with MySQL Server on Windows machines. There has not been any further investigation yet.

**Describe the solution you would like**
We have to test both the docker functionality and slick (DB support) on windows machines. We have to find out how to avoid problems, and add any additional step that needs to be performed on windows to the readme file.

**Comment by sami-cseseu** [comment]
I have checked it in windows it is working great. It is creating the tables and also storing instance. One small problem, it is unable to set default value for timestamp for InstanceEvents.
![error](https://user-images.githubusercontent.com/15995516/49838798-a9d04100-fdac-11e8-8eff-7bc24132cec5.PNG)

**Comment by johannesduesing** [comment]
That's odd, i don't know why it would ever need a default value for the timestamp, since all events have a defined timestamp. But at least you are able to connect to your DB on your system. Which versions of SQL and Windows are you running on ?

**Comment by sami-cseseu** [comment]
I am using MySQL 5.7.19 in windows 10. It is one of the service of my WAMP .

**Comment by sami-cseseu** [comment]
Default timestamp problem in the windows MySQL is also fixed

**Comment by johannesduesing** [comment]
Testing Docker on Windows requires Hyper-V, which is not enabled in Win 10 Home.

**Comment by sami-cseseu** [comment]
I have finally able to install docker. So I checked thoroughly with both type of container. Linux and Windows container.

**For windows container:**
It has a way to define API port. but the _sbt docker:publishlocal_ does not works in the container. It works only on experimantal mode which uses both linux and windows container. It is not very convenient way to run but Instence registry works as it suppose to do.

**For linux container:**
We do not need to use any kind of experimental mode _sbt docker:publishlocal_ works fine everything works but the problem is seting up api route to tcp://0.0.0.0:9095. It does not have any option to change its api port I have tried all the process I can. It only have a option to set the api port at tcp://localhost:2375 if I modify the code use this port all the function works good

What I learned from docker documentation that it suggests to use  tcp://localhost:2375 for its daemon connection we can discuss about whether we can use tcp://localhost:2375 for the connection then it will be much easier to integrate our project in docker windows though there are some other small issues. I am still trying to change the daemon port for linux container if I able to change it I will inform.
